contrib/inkscape_tools/tests/app/src/s2pn.nit

```diff
@@ -22,27 +22,24 @@ redef class App
 
 	init do super
 
-	redef fun on_create
-	do
+	redef fun on_create do
```

Comment:
Fixing whitespaces is fine with me, but please preserve the coding style of the inkscape_tools project. The `do`s should stay on the next line.